src/components/Sections/ItemsSection.js

```diff
@@ -161,18 +161,25 @@ class ItemsSection extends Component {
                   width: colSpan * columnWidth
                 };
                 const type = item.fieldType || '';
-                let dataDisplay = Array.isArray(item.data) ?
-                  <SectionTable data={item.data} /> :
-                  <SectionMarkdown
-                    text={String(item.data)}
-                    markdownArtifactsServerAddress={markdownArtifactsServerAddress}
-                  />;
-                if (type === SECTION_ITEM_TYPE.html) {
-                  dataDisplay = <SectionHTML text={item.data} />;
+                let dataDisplay = item.data;
```

Comment:
i don't think i makes sense. Rather its been `null` to avoid errors when printing objects to react dom

Reply:
it had changed in the last fix. please re-review